docs/assets/documents/governance-for-hospitalNotification.md

```diff
@@ -0,0 +1,111 @@
+<a href="../images/HospitalNotification.svg" target="_blank"> <img src="../images/HospitalNotification.svg" alt="The structure and included profiles in HospitalNotification" style="width:auto; margin-left:0px; margin-right:0px;" id="Fig2"></a>
+<figcaption text-align="left"><b>Figure 2: The structure and included profiles in HospitalNotification</b></figcaption>
+</figure>
+<br>
+
+### 2.1 MedComHospitalNotificationMessage profile
+
+|Rule name|Rules used to constrain the possibilities in a MedComHospitalNotificationMessage profile|
+|:---|:---|
+|medcom-hospitalNotification-1 | The MessageHeader **SHALL** conform to MedComHospitalNotificationMessageHeader |
+|medcom-hospitalNotification-2 | Entry **SHALL** contain exactly one patient resource |
+|medcom-hospitalNotification-3 | All Provenance resources **SHALL** contain activities from medcom-hospitalNotification-messageActivities ValueSet |
+|medcom-hospitalNotification-4 | The system of Patient.identifier **SHALL** be 'urn:oid:1.2.208.176.1.2', which represents an official Danish CPR-number |
+|medcom-hospitalNotification-5 | The receiver of a HospitalNotification **SHALL** be a primary receiver. |
+|medcom-messaging-2 | There **SHALL** be at least one Provenance resource in a MedCom message |
+|medcom-messaging-3 | All Provenance resources shall conform to medcom-messaging-provenance profile |
+
+<br>
+
+### 2.2 MedComHospitalNotificationEncounter profile
+
+One episodeOfCare-identifier **SHALL** be included. 
+
+|Rule name|Rules used to constrain the possibilities in a MedComHospitalNotificationEncounter profile|
+|:---|:---|
+|uuidv5 | The episodeOfCare-identifier **MAY** be an LPR3-identifier, which **SHALL** be an UUID version 5. |
+|medcom-hospitalNotification-6 | When the Encounter.status = onleave, the timestamp Encounter.extension:leavePeriod.start **SHALL** be present|
+
+### 2.3 MedComMessagingProvenance
+
+When sending an initial HospitalNotification exactly one instance of the Provenance resource **SHALL** be included. An initial HospitalNotification can be of the type STIN or STAA, as described in the [Clinical guidelines for application](https://medcomdk.github.io/dk-medcom-hospitalnotification/#11-clinical-guidelines-for-application). 
+
+When sending a following HospitalNotification, both the previous instances describing the previous HospitalNotifications and a new instance describing the current HospitalNotification **SHALL** be included. 
+
+> Note: The lateste Provenance can be identified be comparing Provenance.occurred in across all instances or by identify Provenance.target that is equal to MessageHeader.id in the current message.
+
+When acknowlegding a HospitalNotification with an Acknowledgement, only the latest instance of the Provenance the HospitalNotification and the current instance of the Provenance describing the Acknowledgement **SHALL** be include. Therefore, will an Acknowledgement always include two instances of Provenance resources.
```

Comment:
acknowlegding 
erstattes af
acknowledging 

**SHALL** be include. Therefore, will an Acknowledgement always include two instances of Provenance resources.
erstattes af 
**SHALL** be included. Therefore, an Acknowledgement will always contain two instances of Provenance resources.